### PR TITLE
DOC: Blogpost 9,10,11

### PR DESCRIPTION
**Pull request by deka27**
Blogpost for week 9,10,11

### CODE REVIEW

**Review by itellaetxe — APPROVED** [review]
Very nice blogs Kaustav! Thanks for the update.

Also, keep it up with the PhD process, I hope that you will get accepted.

LGTM!

File: posts/2024/2024_07_29_kaustav_week_9.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+My Journey Continues: Week 9 Progress with DIPY
```
Comment (WassCodeur) [inline posts/2024/2024_07_29_kaustav_week_9.rst:1]: Hi Kaustav!

Great Job and good luck with your PhD applications.

See some comments below

File: posts/2024/2024_07_29_kaustav_week_9.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+I started my Docker journey by exploring the official `Docker documentation <https://docs.docker.com/get-started/>`_, which provided a solid foundation. The tutorials and examples there were incredibly helpful in understanding the basics of Docker, including concepts like containers, images, Dockerfiles, and volumes. The step-by-step guide walked me through setting up my first container, which was a satisfying experience.
+
+**Trial and Error**
+
+While the documentation gave me a good start, much of my learning came through trial and error. I experimented with creating and running containers, exploring how to build custom Docker images, and understanding how to manage multiple containers simultaneously.
+
+I also explored some video tutorials on YouTube.
+
+1. https://www.youtube.com/watch?v=pTFZFxd4hOI&t=551s
```
Comment (WassCodeur) [inline posts/2024/2024_07_29_kaustav_week_9.rst:26]: You could use a Sphinx "inline external link" to reference tutorial links like you did at [L18].

`Docker Tutorial for Beginners <https://www.youtube.com/watch?v=pTFZFxd4hOI&t=551s>`_

File: posts/2024/2024_07_29_kaustav_week_9.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+2. https://www.youtube.com/watch?v=GFgJkfScVNU&t=1432s
```
Comment (WassCodeur) [inline posts/2024/2024_07_29_kaustav_week_9.rst:27]: same here

File: posts/2024/2024_08_12_kaustav_week_11.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+   :author: Kaustav
+   :tags: google
+   :category: gsoc
+
+Hello everyone! Week 11 has been another week of progress, although it came with its own set of challenges. I’ve been working on Docker, making updates to the tutorial fixes, and I also have some exciting news on the personal front. Let me take you through the highlights of this week.
+
+Creating a Docker Image for DIPY
+--------------------------------
+
+This week, I focused on creating a Docker image for DIPY. However, I ran into some issues related to the `fury` and `vtk` packages due to version incompatibilities. These packages are crucial for the visualization components of DIPY, and ensuring they work correctly within the Docker environment has been a bit tricky.
```
Comment (WassCodeur) [inline posts/2024/2024_08_12_kaustav_week_11.rst:14]: `fury` --> `FURY`
`vtk` --> `VTK`

File: posts/2024/2024_08_12_kaustav_week_11.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+The version conflicts have caused some build failures, and I’ve been troubleshooting the issue by testing different configurations and dependencies. While I haven’t completely resolved the issue yet, I’m hopeful that I can find a solution soon. I’ll continue to work on this in the coming days, as getting this Docker image ready is important for our continuous integration processes.
+
+.. image:: /_static/images/gsoc/2024/kaustav/docker_issue_fury.png
+    :alt: Docker Fury VTK issue
```
Comment (WassCodeur) [inline posts/2024/2024_08_12_kaustav_week_11.rst:19]: Fury --> FURY

File: posts/2024/2024_08_12_kaustav_week_11.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+On a personal note, I successfully submitted my PhD application this week! This has been a significant milestone for me, and I’m thrilled to have it completed. Applying for a PhD is a rigorous process, and I’ve put a lot of effort into crafting the application. Now that it’s submitted, I’m hopeful for the best outcome.
+
+If I do get accepted, it would be a dream come true. I’m particularly excited about the prospect of learning from and working with leaders in the field, such as `Dr. Eleftherios Garyfallidis <https://github.com/Garyfallidis>`__, whose work I greatly admire. The opportunity to contribute to and learn from such a program would be invaluable, and I’m eagerly looking forward to the possibility.
+
+Next Week
+---------
+
+Looking ahead to next week, my priorities will be:
+
+1. **Fixing the Docker Image Issues**: I’ll continue troubleshooting the `fury` and `vtk` package issues within the Docker environment, aiming to get a working Docker image for DIPY.
```
Comment (WassCodeur) [inline posts/2024/2024_08_12_kaustav_week_11.rst:41]: `fury` --> `FURY`
`vtk` --> `VTK`

File: posts/2024/2024_07_29_kaustav_week_9.rst [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+For reference, I was pointed to look at the pyAFQ docker as well. (https://github.com/yeatmanlab/pyAFQ)
+
+**Docker in the Context of DIPY**
+
+The reason for learning Docker was for creation of DIPY docker image that can be utilized easily for running DIPY and doing tests.
+
+PhD Applications
```
Comment (robinroy03) [inline posts/2024/2024_07_29_kaustav_week_9.rst:35]: atb :)

**Review by robinroy03 — APPROVED** [review]

File: posts/2024/2024_07_29_kaustav_week_9.rst (hunk 7 above)
Comment (deka27) [reply posts/2024/2024_07_29_kaustav_week_9.rst:35]: Thanks

File: posts/2024/2024_07_29_kaustav_week_9.rst (hunk 1 above)
Comment (deka27) [reply posts/2024/2024_07_29_kaustav_week_9.rst:1]: Thank you.

**Review by skoudoro — APPROVED** [review]
Àll good, Thank you @deka27, merging

**Pull request merged**